src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2814,6 +2826,7 @@ components:
     # /stream/* Request & Response Types
     #
 
+    # TODO PP: update docs with examples.
```

Comment:
Heads up - TODO still here - if it was intended to be done now? If not, we should raise a ticket for it and put the ticket reference here.

Reply:
i wanted to complete it once it passes review :P I was kind of sure that name of props might change.